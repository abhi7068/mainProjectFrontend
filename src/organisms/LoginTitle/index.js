import React from 'react';

import { makeStyles } from '@material-ui/core/styles';
// import LoginImage from '../../atoms/LoginImage';
import logoMob from '../../images/yoyo-logo-mob.png';

import TitleComponent from '../../atoms/TitleComponent';
import Subtitle from '../../atoms/Subtitle';

// oganism styling, positioning atoms on layout
const useStyles = makeStyles((theme) => ({
  leftTitle: {
    position: 'relative',
    width: '100%',
    height: '100%',
  },
  positionImage: {
    position: 'absolute',
    bottom: '0',
    left: '50%',
    transform: 'translateX(-50%)',
  },
}));

// Left side of login page
function LoginTitle() {
  const classes = useStyles();
  return (
    <div className={classes.leftTitle}>
      <TitleComponent>Login</TitleComponent>
      <Subtitle>
        <span>One step away to send gifts to your loved ones</span>
      </Subtitle>
      <div className={classes.positionImage}>
        {/* <LoginImage /> */}
        <img src={logoMob} width={200} alt="login" />
      </div>
    </div>
  );
}

export default LoginTitle;
