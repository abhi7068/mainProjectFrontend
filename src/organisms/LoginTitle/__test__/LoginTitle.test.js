import React from 'react';

import LoginTitle from '../index';
import { render } from '@testing-library/react';

it('it renders', () => {
  render(<LoginTitle />);
});

it('should take a snapshot', () => {
  const { asFragment } = render(<LoginTitle />);
  expect(asFragment(render(<LoginTitle />))).toMatchSnapshot();
});
