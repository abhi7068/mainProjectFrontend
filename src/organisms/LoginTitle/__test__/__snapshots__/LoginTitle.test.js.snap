// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-leftTitle-34"
  >
    <h2
      class="MuiTypography-root MuiTypography-h5"
      data-testid="Typography"
      style="font-size: 29px; font-weight: 500;"
    >
      Login
    </h2>
    <p
      class="makeStyles-subtitle-66"
    >
      <span>
        One step away to send gifts to your loved ones
      </span>
    </p>
    <div
      class="makeStyles-positionImage-35"
    >
      <img
        alt="login"
        src="yoyo-logo-mob.png"
        width="200"
      />
    </div>
  </div>
</DocumentFragment>
`;
